config/initializers/CarrierWave.rb

```diff
@@ -0,0 +1,15 @@
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+    provider: 'AWS',
+    aws_access_key_id: ENV["AWS_ACCESS_KEY_ID"],
+    aws_secret_access_key: ENV["AWS_SECRET_ACCESS_KEY"]
+    }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.